meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -0,0 +1 @@
+MACHINE_FEATURES_append_ls1021atwr += "alsa"
```

Comment:
Please don't combine _append and +=, they're both append operations. Instead, make the separator explicit:
```
MACHINE_FEATURES_append_ls1021atwr = " alsa"
```
Also, we should add a comment above this. It will show up in the user's local.conf, so we should at least make a best effort to explain why we're overriding/altering things the machine normally takes care of at that level.

Other than that, this looks great, thanks.